### PR TITLE
Editorial update

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

**Review by shargon — APPROVED** [review]

**Review by AnnaShaleva — APPROVED** [review]

**Review by Jim8y — APPROVED** [review]

File: nep-1.mediawiki [hunk 1]
```diff
@@ -140,7 +140,7 @@ If you are interested in assuming ownership of a NEP, send a message asking to t
 
 The current NEP editor is
 
-*Erik Zhang (@erikzhang)
+*Fernando Díaz Toledano (@shargon)
```
Comment (vncoelho) [inline nep-1.mediawiki:143]: Why this change?

Comment (vncoelho) [reply nep-1.mediawiki:143]: I see now, the editor.

**Review by vncoelho — APPROVED** [review]
